### DISCUSSION TITLE
Cannot log in

### DISCUSSION THREAD

**Discussion by Ninoo2144** [vercel/community#3208] [Help] [answered]
I cannot log in or sign up with my github account on vercel.
When I try it says: This user's account has been blocked.
I recived 0 emails about this situation and I need help.

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
